servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/HttpsProxyTest.java

```diff
@@ -153,6 +153,7 @@ void tearDown() throws Exception {
         safeClose(proxyTunnel);
     }
 
+    @SuppressWarnings("PMD.AvoidPrintStackTrace")
```

Comment:
Is this necessary? It seems to do the right thing.

Reply:
Good catch, I did these pmd changes long time ago, found out on the plain that I forgot to push them. Some other change switched from print-stack-trace to a logger. Not necessary to have this suppression anymore.